### PR TITLE
Task-#37

### PR DESCRIPTION
**Pull request by tjjohn1**
Changes made for previous language differences, changed to English from Spanish in the following locations:
        AgendaGenerator.java - Line 380
    ExportSticker.java - Lines 72,77
    AgendaPanel.java - Lines 51,195,201,206

### CODE REVIEW

File: src/net/sf/memoranda/ui/AgendaPanel.java [hunk 1]
```diff
@@ -48,7 +48,7 @@
 	JButton historyForwardB = new JButton();
 	JButton export = new JButton();
 	JEditorPane viewer = new JEditorPane("text/html", "");
-	String[] priorities = {"Muy Alta","Alta","Media","Baja","Muy Baja"};
+/*TJ*/	String[] priorities = {"Very High","High","Medium","Low","Very Low"};
```
Comment (ghgofort) [inline src/net/sf/memoranda/ui/AgendaPanel.java]: Language changes reviewed by ghgofort @12:21am 2/8/16

File: src/net/sf/memoranda/ui/AgendaPanel.java [hunk 2]
```diff
@@ -190,20 +190,20 @@ else if (d.startsWith("memoranda:tasks")) {
...
+						 /*  You need to add the export sticker meanwhile..*/
 						 final JFrame parent = new JFrame();
-						 String name = JOptionPane.showInputDialog(parent,Local.getString("Ingrese nombre de archivo a exportar"),null);
+/*TJ*/						 String name = JOptionPane.showInputDialog(parent,Local.getString("Enter file name to export"),null);
 						 new ExportSticker(name).export("html");
 						 //JOptionPane.showMessageDialog(null,name);
 					}else if (d.startsWith("memoranda:importstickers")) {
 						final JFrame parent = new JFrame();
-						String name = JOptionPane.showInputDialog(parent,Local.getString("Ingrese nombre de archivo a importar"),null);
+/*TJ*/						 String name = JOptionPane.showInputDialog(parent,Local.getString("Enter file name to import"),null);
```
Comment (ghgofort) [inline src/net/sf/memoranda/ui/AgendaPanel.java:206]: Language changes reviewed by ghgofort 12:23am - 2/8/16.  All changes look ready to commit.

File: src/net/sf/memoranda/ui/ExportSticker.java [hunk 3]
```diff
@@ -69,12 +69,12 @@ public boolean export(String src){
...
                         fwrite.close();
-                        JOptionPane.showMessageDialog(null,Local.getString("Documento creado con exito en su carpeta Memoranda =D"));
+/*TJ*/                        JOptionPane.showMessageDialog(null,Local.getString("Document created with success in its Memoranda folder =D"));
             
             
         } catch (IOException e) {
             e.printStackTrace();
-            JOptionPane.showMessageDialog(null,Local.getString("NO Logramos crear su documento =(..."));
+/*TJ*/            JOptionPane.showMessageDialog(null,Local.getString("We failed to create your document =(..."));
         }
```
Comment (ghgofort) [inline src/net/sf/memoranda/ui/ExportSticker.java:78]: Language changes reviewed by ghgofort 12:23am - 2/8/16. All changes look ready to commit.

File: src/net/sf/memoranda/util/AgendaGenerator.java [hunk 4]
```diff
@@ -377,7 +377,7 @@ public static String generateStickers(CalendarDate date) {
 				.class
 				.getResource("resources/agenda/removesticker.gif")
 				.toExternalForm();
-		 String s = "<hr><hr><table border=\"0\" cellpadding=\"0\" width=\"100%\"><tr><td><a href=\"memoranda:importstickers\"><b>"+Local.getString("Importar anotaci�n")+"</b></a></td><td><a href=\"memoranda:exportstickerst\"><b>"+Local.getString("Exportar anotaci�n como .txt")+"</b></a><td><a href=\"memoranda:exportstickersh\"><b>"+Local.getString("Exportar anotaci�n como .html")+"</b></a></td></tr></table>"
+/*TJ*/		 String s = "<hr><hr><table border=\"0\" cellpadding=\"0\" width=\"100%\"><tr><td><a href=\"memoranda:importstickers\"><b>"+Local.getString("Import annotation")+"</b></a></td><td><a href=\"memoranda:exportstickerst\"><b>"+Local.getString("Export annotation as .txt")+"</b></a><td><a href=\"memoranda:exportstickersh\"><b>"+Local.getString("Export annotation as .html")+"</b></a></td></tr></table>"
 				 +   "<table border=\"0\" cellpadding=\"0\" width=\"100%\"><tr><td><a href=\"memoranda:addsticker\"><img align=\"left\" width=\"22\" height=\"22\" src=\""
```
Comment (ghgofort) [inline src/net/sf/memoranda/util/AgendaGenerator.java:381]: Language changes reviewed by ghgofort 12:23am - 2/8/16. All changes look ready to commit.

File: src/net/sf/memoranda/ui/AgendaPanel.java (hunk 1 above)
Comment (ghgofort) [inline src/net/sf/memoranda/ui/AgendaPanel.java:51]: Language changes reviewed by ghgofort 12:23am - 2/8/16. All changes look ready to commit.

**Comment by tjjohn1** [comment]
Cool thanks!

**Pull request merged**